docs/design/README.md

```diff
@@ -0,0 +1,46 @@
+# Design of Node Controller Manager
+
+The design of the Node Controller Manager is influenced by the Kube Controller Manager, where-in mutiple sub-controllers are used to manage the kubernetes clients.
```

Comment:
s/mutible/multiple/

Reply:
It already says multiple right ?

Reply:
I still see "where-in mutible sub-controllers" in this sentence, should be replaced by "multiple"

Reply:
I still see it as multiple. I crossed checked with Hardik as well. Are we missing something here ?